packages/pluggableWidgets/datagrid-web/src/components/Table.tsx

```diff
@@ -13,16 +13,22 @@ import {
     SortingRule,
     useColumnOrder,
     useFilters,
-    useFlexLayout,
     usePagination,
-    useResizeColumns,
     useSortBy,
     useTable
 } from "react-table";
 import { ColumnsPreviewType, ColumnsType, FilterMethodEnum } from "../../typings/DatagridProps";
+import ReactResizeDetector from "react-resize-detector";
 
 export type TableColumn = Omit<ColumnsType | ColumnsPreviewType, "content" | "attribute">;
 
+export interface HeaderSize {
+    [key: string]: {
+        resized: boolean;
```

Comment:
You don't seem to set this flag anywhere

Reply:
Just a preparation for the fix in WT-2790 for the incoming sprint